src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -93,31 +93,31 @@ public with sharing class AFFL_MultiRecordTypeMapper {
         // Map of Account RecordType Label To API
         accRecTypeLabelToAPI = getAccRecTypeLabelToAPIMap();
         
-        //List of primary affiliation lookup fields labels.
-        List<String> primaryAfflFieldLabels = accRecTypeToContactLabel.values();
+        //List of primary affiliation lookup fields labels/API names from settings.
+        List<String> primaryAfflFieldsFromSettings = accRecTypeToContactLabel.values();
```

Comment:
These can be either label names or API names, so changing the name @xinnanlisf . Just changed to 'accRecTypeToContactLabel' to 'primaryAfflFieldInSettingsByAccRecType' because it could either be API names or Labels not just labels.